docs/running.rst

```diff
@@ -271,6 +271,51 @@ Typing ``python toil-sort-example.py --help`` will show the complete list of arg
 both Toil's and ones defined inside **toil-sort-example.py**.  A complete explanation of Toil's arguments can be found
 in :ref:`commandRef`.
 
+Environment Variable Options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+There are several environment variables that affect the way Toil runs.
+
+TOIL_WORKDIR: An absolute path to a directory where Toil will write its
+temporary files.
+
+TOIL_TEST_TEMP: An absolute path to a directory where Toil tests will
+write their temporary files.
+
+TOIL_RT_LOGGING_ADDRESS: In the form IP:Port, this is the address to use
+for Toil's realtime logging.
```

Comment:
When does the user need to set this? If this is purely internal, I would mention this variable here.